### PR TITLE
Cypress fix: resizing mizu when needed

### PR DESCRIPTION
**Pull request by AdamKol-up9**

### CODE REVIEW

File: acceptanceTests/cypress/integration/testHelpers/TrafficHelper.js [hunk 1]
```diff
@@ -82,27 +84,38 @@ export function checkFilterByMethod(funcDict) {
...
-                            if (summaryDict)
-                                leftTextCheck(entryNum, summaryDict.pathLeft, summaryDict.expectedText);
-
-                            if (!doneCheckOnFirst) {
-                                deepCheck(funcDict, protocolDict, methodDict, entry);
-                                doneCheckOnFirst = true;
+                    cy.get('#entries-length').invoke('text').then(len => {
+                        resizeIfNeeded(len);
+                        listElmWithIdAttr.forEach(entry => {
+                            if (entry?.id && entry.id.match(RegExp(/entry-(\d{2}|\d{1})$/gm))) {
```
Comment (gadotroee) [inline acceptanceTests/cypress/integration/testHelpers/TrafficHelper.js:90]: if you know it is going to be number - there is no need in regex,
you can probably convert it to int and do some operation like > < and =

File: acceptanceTests/cypress/integration/tests/Rabbit.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import {checkFilterByMethod, valueTabs,} from "../testHelpers/TrafficHelper";
 
-
 it('opening mizu', function () {
     cy.visit(Cypress.env('testUrl'));
+    cy.get('#total-entries').invoke('text').should('match', /^[4-7][0-9]$/m)
```
Comment (gadotroee) [inline acceptanceTests/cypress/integration/tests/Rabbit.js:5]: same here, convert to number and do >= 40 and < 80 or something like this

**Review by gadotroee — CHANGES_REQUESTED** [review]
added two improvement suggestions

File: acceptanceTests/cypress/integration/testHelpers/TrafficHelper.js (hunk 1 above)
Comment (AdamKol-up9) [reply acceptanceTests/cypress/integration/testHelpers/TrafficHelper.js:90]: yes i thought its prettier with regex, but I'll change it

Comment (AdamKol-up9) [reply acceptanceTests/cypress/integration/testHelpers/TrafficHelper.js:90]: the regex is for checking  that the id is -> 'entry-X', its not only a number

File: acceptanceTests/cypress/integration/tests/Rabbit.js (hunk 2 above)
Comment (AdamKol-up9) [reply acceptanceTests/cypress/integration/tests/Rabbit.js:5]: I need to use here the should(), If I convert the entries number to int I will need to use expect() instead, which is not good for this case.

**Review by gadotroee — APPROVED** [review]

**Pull request merged**